lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -47,11 +50,13 @@ class ExampleWording < Base
         extend AutoCorrector
 
         MSG_SHOULD = 'Do not use should when describing your tests.'
+        MSG_WILL   = 'Do not use the future tense when describing your tests.'
         MSG_IT     = "Do not repeat 'it' when describing your tests."
         MSG_INSUFFICIENT_DESCRIPTION = 'Your example description is ' \
                                        'insufficient.'
 
         SHOULD_PREFIX = /\Ashould(?:n't)?\b/i.freeze
+        WILL_PREFIX   = /\Aw(?:ill|on't)\b/i.freeze
```

Comment:
I think there’s nothing gained by isolating the `w`, but it becomes harder to read.
```suggestion
        WILL_PREFIX   = /\A(?:will|won't)\b/i.freeze
```